rust/ast-generator/src/main.rs

```diff
@@ -487,6 +487,9 @@ fn main() -> anyhow::Result<()> {
 
     grammar.enums.retain(|x| x.name != "Adt");
 
+    // we flatten TypeAnchor into PathSegment in the extractor
+    grammar.nodes.retain(|x| x.name != "TypeAnchor");
```

Comment:
Hopefully they give the other types of path segments a name too at some point so it becomes a usable enum.